packages/core/test/input.test.ts

```diff
@@ -198,7 +198,7 @@ describe('input', () => {
       {
         invoke: {
           src: 'child',
-          input: ({ event }) => {
+          input: ({ event }: any) => {
```

Comment:
See #4054 - that would remove the need for this

Reply:
I know. That's why I didn't even bother typing this here. It will be an easy conflict resolution either way.